app/policies/admin/api_key_rubygem_scope_policy.rb

```diff
@@ -0,0 +1,11 @@
+class Admin::ApiKeyRubygemScopePolicy < Admin::ApplicationPolicy
+  class Scope < Scope
```

Comment:
Is this code right? It's hard to tell what namespaces are these 2 classes working under, maybe we need to add a comment to make it clear?

Reply:
I updated this to be more explicit:

```
  class Scope < Admin::ApplicationPolicy::Scope
```